src/Writer/MarkdownWriter.php

```diff
@@ -47,18 +49,20 @@ public function createDocumentation(string $filepath, array $installedPackages,
                 }
             }
 
-            // @TODO: Maybe add documentation for packages who were documented but not installed (anymore)
-
             $documentation[] = "";
+        }
 
-            $handle = @fopen($filepath, "w");
+        // @TODO: Maybe add documentation for packages who were documented but not installed (anymore)
 
-            foreach ($documentation as $line) {
-                // @TODO: which line break?!
-                fwrite($handle, "$line\r\n");
-            }
+        $documentation[] = "";
 
-            fclose($handle);
+        $handle = @fopen($filepath, "w");
+
+        foreach ($documentation as $line) {
+            // @TODO: which line break?!
+            fwrite($handle, "$line\r\n");
```

Comment:
I know, not a concern of this issue, but can't we just use PHP_EOL here? Eh, there's probably a better way to write files anyway, we'll fix this later!